src/plugin/xyz/xyzdescriptornodeui.js

```diff
@@ -90,6 +92,14 @@ class Controller extends DescriptorNodeUICtrl {
     };
     uiWindow.create(windowOptions, editUi, undefined, undefined, undefined, scopeOptions);
   }
+
+  /**
+   * @override
+   * @return {ConfigDescriptor}
+   */
+  getDescriptor() {
+    return /** @type {ConfigDescriptor} */ (super.getDescriptor());
```

Comment:
Bit of a weird override, is this cleaner than casting the return value where its called in here?

Reply:
I can't recall why I thought this was a good idea. Replaced with a cast on the call.